spec/integration/triggers.feature

```diff
@@ -0,0 +1,66 @@
+Feature: triggers
+
+  Creating triggers, editing source and target branches, changing workflows,
+  reordering triggers, switching between trigger types should be possible.
+
+  Background:
+    Given triggers tab is open
+
+  Scenario: Clicking through filters
+    When I click on "Tag filter"
+    Then I should see 0 triggers
+    When I click on "Pull Request filter"
+    Then I should see a trigger
+    When I click on "Push filter"
+    Then I should see 2 triggers
+
+  Scenario: Create trigger but remove it on Cancel click
+    When I click on "Tag filter"
+    Then I should see 0 triggers
+      And "Save Button" should "be disabled"
+    When I click on "Add trigger button"
+    Then I should see a trigger
+      And "Save Button" should "not be disabled"
```

Comment:
This is weird, but yeah this is how it works currently: an add-in-progress trigger is already pushed into the bitrise.yml in the background.